testing/test_collection.py

```diff
@@ -1404,3 +1404,18 @@ def a(): return 4
     result = testdir.runpytest()
     # Not INTERNAL_ERROR
     assert result.ret == ExitCode.INTERRUPTED
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="win32 has no recursive symlinks")
```

Comment:
I think the error is only different / needs other handling.
This is how pathlib does it: https://github.com/python/cpython/commit/d5c120f7eb6f2a9cdab282a5d588afed307a23df